tests/query/test_query.py

```diff
@@ -6,19 +6,12 @@
 def test_empty_query():
     query = Query({}, TableSource("my_table", ColumnSet([])))
 
-    assert query.get_selected_columns() is None
```

Comment:
I don't think this is really useful. In the old representation you initialized the query with a dictionary and the accessors looked for keys in the dictionaries and applied defaults. This was the goal of the test. This does not really apply to the ast, since all fields are initialized to None by the initializer